lib/api/controllers/collectionController.js

```diff
@@ -67,6 +67,9 @@ class CollectionController extends BaseController {
    */
   updateMapping(request) {
     assertHasBody(request);
+
+    // @todo indexCache exists + add in controller
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)